portfolio/src/main/webapp/script.js

```diff
@@ -40,27 +40,34 @@ function getBlogComments() {
 
             console.log(commentsList);
 
-            for (let i = 0; i < commentsList.length; i++) {
+            if(commentsList.length == 0) {
+                var noComments = document.createElement("p");
+                noComments.innerText = "Be the first to comment!";
 
-                var comment = document.createElement("div");
-                comment.classList.add("comment-card");
-                comment.innerHTML = createCommentElement(commentsList[i]);
-
-                commentsDiv.appendChild(comment);
+                commentsDiv.appendChild(noComments);
             }
+
+            else {
```

Comment:
Blank line between if and else, It might be better to have else on same line as closing brace

Reply:
You're right, that would work better. I'll make that change. 👍